routes/admin/authenticated/sidebar/community-settings/domain/page.js

```diff
@@ -97,62 +110,130 @@ class Page extends Component {
           />
           {this.state.deletedHostedZone && (
             <Dialog
               onConfirm={() => this.confirmDeleteDomain()}
               onCancel={() => this.setState({ deletedHostedZone: undefined })}
             >
-              <p>Tem certeza que deseja remover o domínio <b>{this.state.deletedHostedZone.domain_name}</b>?</p>
+              <p>
+                <FormattedMessage
+                  id='community.page--domain-list.dialog.domain-confirm-message'
+                  defaultMessage='Tem certeza que deseja remover o domínio'
+                />
+                <b> {this.state.deletedHostedZone.domain_name}</b>?</p>
             </Dialog>
           )}
         </div>
         {this.state.dnsHostedZone ? (
-          <div className='dns-records'>
-            <h2>Subdomínios externos</h2>
-            {dnsRecordsIsLoading && <Loading />}
-            {this.state.dnsRecords.map((dnsRecord, index) => (
-              <SubdomainPreview
-                key={`dns-record-${index}`}
-                subdomain={dnsRecord}
-                menuComponent={this.dnsRecordMenu(dnsRecord)}
-              />
-            ))}
-            {this.state.showSubdomainForm ? (
-              <SubdomainForm
-                dnsHostedZone={this.state.dnsHostedZone}
-                onSubmit={values => {
-                  const name = `${values.name}.${this.state.dnsHostedZone.domain_name}`
-                  return createDNSRecord({
-                    ...values,
-                    name,
-                    ttl: 3600,
-                    dns_hosted_zone_id: this.state.dnsHostedZone.id
-                  })
-                  .then(dnsRecord => {
-                    this.setState({ dnsRecords: [...this.state.dnsRecords, dnsRecord], showSubdomainForm: false })
-                    this.props.resetForm()
-                    return Promise.resolve()
-                  })
-                }}
-                {...formProps}
-              />
-            ) : (
-              <ButtonPreview
-                text='Adicionar novo subdomínio externo'
-                onClick={() => this.setState({ showSubdomainForm: true })}
-              />
-            )}
-            {this.state.deletedDNSRecord && (
-              <Dialog
-                onConfirm={() => this.confirmDeleteSubdomain()}
-                onCancel={() => this.setState({ deletedDNSRecord: undefined })}
-              >
-                <p>Tem certeza que deseja remover o subdomínio <b>{this.state.deletedDNSRecord.value}</b>?</p>
-              </Dialog>
-            )}
+          <div className='dns-detail'>
+            <div className='dns-records'>
+              <h3>
+                <FormattedMessage
+                  id='community.page--domain-list.header.dns-records'
+                  defaultMessage='Registros DNS'
+                />
+              </h3>
+              {dnsRecordsIsLoading && <Loading />}
+              <p>
+                <FormattedHTMLMessage
```

Comment:
@igr-santos, according to [React-intl documentation](https://github.com/yahoo/react-intl/wiki/Components#formattedhtmlmessage), the `<FormattedHTMLMessage />` is not recommended. Instead using it, it would be better do something like below:

**Current**
```jsx
<FormattedHTMLMessage
  id="community.page--domain-list.dns-record-description"
  defaultMessage={`
    Os registros DNS são configurações especiais que alteram a
    forma como o seu domínio trabalha. Com esses registros, você
    se conecta a serviços de terceiros como provedores de email.
    <a href='https://trilho.bonde.org' title='Saiba mais' target='_blank'>Saiba mais</a>.
  `}
/>
```

**Without FormattedHTMLMessage component**
```jsx
<FormattedMessage
  id="community.page--domain-list.dns-record-description"
  defaultMessage={
    'Os registros DNS são configurações especiais que alteram a' +
    'forma como o seu domínio trabalha. Com esses registros, você' +
    'se conecta a serviços de terceiros como provedores de email. {link}.'
  }
  values={{
    link: (
      <a href='https://trilho.bonde.org' title='Saiba mais' target='_blank'>
        <FormattedMessage
          id="community.page--domain-list.dns-record-description.link"
          defaultMessage="Saiba mais"
        />
      </a>
    )
  }}
/>
```